### PR TITLE
Portal fix

### PR DESCRIPTION
**Pull request by akinsey**
# Merge core pg PR too
 
Portal feature was broken due to broken query

### CODE REVIEW

File: modules/ept-frontend/client/admin/settings/general.html [hunk 1]
```diff
@@ -51,6 +51,9 @@ <h5 class="thin-underline section-header-top-spacing">Portal Configurations</h5>
     </div>
   </div>
 
+  <!-- MOTO -->
```
Comment (unenglishable) [inline modules/ept-frontend/client/admin/settings/general.html]: *MOTD

File: modules/ept-frontend/client/scss/ept/views/layout/_header.scss [hunk 2]
```diff
@@ -558,4 +557,19 @@ header {
...
+    margin-bottom: $breadcrumbs-bottom-margin;
+    #motd {
+      @include outer-container;
+      clear: both;
+      padding: 0.5rem 0;
+      color: #666;
+    }
+  }
 }
+
```
Comment (unenglishable) [inline modules/ept-frontend/client/scss/ept/views/layout/_header.scss]: don't need this line

Comment (unenglishable) [inline modules/ept-frontend/client/scss/ept/views/layout/_header.scss]: but keep newline at the end of the file

**Comment by unenglishable** [comment]
code seems to work in integration, just waiting on the tiny changes i requested

File: modules/ept-frontend/client/scss/ept/views/layout/_header.scss (hunk 2 above)
Comment (akinsey) [inline modules/ept-frontend/client/scss/ept/views/layout/_header.scss]: huh?

**Pull request merged**